app/src/main/kotlin/io/eugenethedev/taigamobile/data/repositories/TasksRepository.kt

```diff
@@ -27,10 +27,11 @@ class TasksRepository @Inject constructor(
         type = statusType
     )
 
-    override suspend fun getFiltersData(commonTaskType: CommonTaskType) = withIO {
+    override suspend fun getFiltersData(commonTaskType: CommonTaskType, isCommonTaskFromBacklog: Boolean) = withIO {
         taigaApi.getCommonTaskFiltersData(
-            CommonTaskPathPlural(commonTaskType),
-            currentProjectId
+            taskPath = CommonTaskPathPlural(commonTaskType),
+            project = currentProjectId,
+            milestone = if (isCommonTaskFromBacklog) "null" else null
```

Comment:
And write something like this here:
```kotlin
milestone = sprintId ?: "null"
```

Reply:
But I don't need `milestone` only in backlog... If I write a `sprintId: Long? = null`, then I will have to add the current sprint everywhere

Reply:
Ok, I just thought that it would look better since in the future we could add support for filters on the sprint screen. But let's leave it like this for now, I guess...